nixos/modules/services/networking/bee.nix

```diff
@@ -55,12 +60,14 @@ in {
 
     warnings = optional (! config.services.bee-clef.enable) "The bee service requires an external signer. Consider setting `config.services.bee-clef.enable` = true";
 
+    # TODO these are supposed to be defaults, but they conflicts with stuff that is set in the user's config.
     services.bee.settings = {
```

Comment:
@SuperSandro2000 these clash with the user's settings, but the defaults that are installed above in the options section don't show up in cfg.settings. i.e. iunless the user sets password-file, there'll be an error that it's undefined.

i need to install defaults for these values that the user can override.

Reply:
You want something like this:
```
services.bee.settings = {
  data-dir = "/var/lib/bee"; # user can overwrite with services.bee.settings.data-dir = lib.mkForce "/custom/dir";
  password-file = lib.mkDefault "/var/lib/bee/password"; # user can overwrite with services.bee.settings.password-file = "/custom/dir/password";
  # etc...
};
```